test/cluster/lwt/test_lwt_during_tablets_migration.py

```diff
@@ -0,0 +1,303 @@
+
+    async def prepare_statements(self):
+        """Prepare CQL statements for row selection and updates"""
+        self.ps_select_row = self.cql.prepare(
+            f"SELECT {self.select_cols} FROM {self.ks}.{self.tbl} WHERE pk = ?"
+        )
+
+        for i in range(self.num_workers):
+            others = self.others_per_worker[i]
+            cond = " AND ".join([f"s{j} >= ?" for j in others] + [f"s{i} = ?"])
+            query = (
+                f"UPDATE {self.ks}.{self.tbl} SET s{i} = ? "
+                f"WHERE pk = ? IF {cond}"
+            )
+            self.ps_update[i] = self.cql.prepare(query)
+
+    async def create_schema(self):
+        """Create table with multiple counter columns"""
+        cols_def = ", ".join(f"s{i} int" for i in range(self.num_workers))
+        await self.cql.run_async(
+            f"CREATE TABLE {self.ks}.{self.tbl} (pk int PRIMARY KEY, {cols_def})"
+        )
+        logger.info(f"Created table {self.ks}.{self.tbl} with {self.num_workers} columns")
+
+    async def initialize_rows(self):
+        """Initialize the test row with all columns set to 0"""
+        zeros = ", ".join("0" for _ in range(self.num_workers))
+        for pk in self.pks:  # self.pks = [1..NUM_KEYS]
+            await self.cql.run_async(
+                f"INSERT INTO {self.ks}.{self.tbl} "
+                f"(pk, {self.select_cols}) VALUES ({pk}, {zeros})"
+            )
+
+    async def _worker(self, worker_id: int):
+
+        rng = self.rngs[worker_id]
+        backoff = BACKOFF_BASE
+
+        while not self.stop_event.is_set():
+            pk = rng.choice(self.pks)
+            if self.stop_event.is_set():
+                break
+            verify_query = self.ps_select_row.bind([pk])
+            verify_query.consistency_level = ConsistencyLevel.LOCAL_QUORUM
+            rows = await self.cql.run_async(verify_query)
+            if not rows:
+                # do we need to handle this case?
+                await asyncio.sleep(backoff); backoff = min(backoff * 2, 0.5); continue
+
+            row = rows[0]
+            prev_val = getattr(row, f"s{worker_id}")
+            new_val = prev_val + 1
+
+            others = self.others_per_worker[worker_id]
+            if_vals = [getattr(row, f"s{col_idx}") for col_idx in others] + [prev_val]
+            params = [new_val, pk] + if_vals
+
+            update = self.ps_update[worker_id].bind(params)
+            update.consistency_level = ConsistencyLevel.LOCAL_QUORUM
+            update.serial_consistency_level = ConsistencyLevel.LOCAL_SERIAL
+
+            try:
+                res = await self.cql.run_async(update)
+                applied = bool(res and res[0].applied)
+            except (WriteTimeout, OperationTimedOut) as e:
+                self._log_entry(worker_id, "fail", error=str(e), pk=pk)
+                verify_after_cas_error = self.ps_select_row.bind([pk])
+                verify_after_cas_error.consistency_level = ConsistencyLevel.LOCAL_QUORUM
```

Comment:
this must be `verify_after_cas_error.consistency_level = ConsistencyLevel.LOCAL_SERIAL` -- we need paxos read for verify.